applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -77,6 +93,8 @@ BUILD_ASSERT(!IS_ENABLED(CONFIG_DESKTOP_HID_STATE_ENABLE) ||
 	     IS_ENABLED(CONFIG_DESKTOP_USB_SELECTIVE_REPORT_SUBSCRIPTION) ||
 	     (ARRAY_SIZE(usb_hid_device) <= 1));
 
+static struct usbd_contex *usbd_ctx;
```

Comment:
Hi @jfischer-no, @tmon-nordic
Could you explain the naming scheme here please? I guess there is a reason why it's called `usbd_contex`

Reply:
`static struct usbd_ctx *usbd_ctx;` is wrong, there is no `struct usbd_ctx` defined anywhere.

The struct is called `struct usbd_contex`. If you want to rename it (I prefer `struct usbd_context`), then the rename should first go upstream.

Reply:
The question is: Do you recall why it's currently called `usbd_contex` in upstream? It seems a bit like a typo, but I guess there is a reason why it's named like that.

Reply:
This is just a typo and the only reason it stays is that it needs a rather big PR to change it everywhere. But if you send PR upstream, I believe it can go in rather quickly (and we could cherry-pick it to NCS as nrf fromtree).

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/73463